src/routes/profile/+page.svelte

```diff
@@ -63,8 +63,48 @@
 </main>
 
 <style>
+  @import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;1,100;1,200;1,300;1,400&display=swap');
```

Comment:
Sebetulnya tidak perlu import font dari gogole, karena font tersebut sudah saya import di file `app.html`, but other than that, good job.